static/app/components/tables/gridEditable/styles.tsx

```diff
@@ -185,11 +185,15 @@ export const GridHeadCellStatic = styled('th')`
 /**
  * GridBody are the collection of elements that contains and display the data
  * of the Grid. They are rather simple.
+ *
+ * If the header is sticky, the body should be layered underneath it.
  */
-export const GridBody = styled('tbody')`
+export const GridBody = styled('tbody')<{stickyHeader?: boolean}>`
   display: grid;
   grid-template-columns: subgrid;
   grid-column: 1/-1;
+
+  ${p => (p.stickyHeader ? `z-index: ${Z_INDEX_STICKY_HEADER - 1}` : '')}
```

Comment:
Out of curiosity, does anything bad happen if this `z-index` is _always_ applied? It'd be nice if people didn't have to know to add `stickyHeader` to the body component